webpack.renderer.config.js

```diff
@@ -46,7 +46,7 @@ const rendererConfig = {
             loader: `url-loader`,
             query: {
               limit: 10000,
-              name: `imgs/[name].[ext]`
+              name: `images/[name].[ext]`
```

Comment:
i don't know how / if this was working before - but `imgs` wasn't an actual path

Reply:
webpack was moving the files in the imgs folder, but this renaming is confusing and does not add value at all 👍 good cleanup

Reply:
<img width="506" alt="screenshot 2019-02-01 at 19 42 35" src="https://user-images.githubusercontent.com/1656471/52145668-937e8d00-2659-11e9-82e6-36e5fa067855.png">